usr/lib/common/pkey_utils.c

```diff
@@ -234,27 +234,30 @@ static int s390_kmac(unsigned long func, void *param,
 
 /**
  * s390_pcc:
- * @func: the function code passed to KM; see s390_pcc_functions
+ * @func: the function code passed to PCC; see s390_pcc_functions
  * @param: address of parameter block; see POP for details on each func
  *
  * Executes the PCC operation of the CPU.
  *
- * Returns -1 for failure, 0 for the query func, number of processed
- * bytes for encryption/decryption funcs
+ * Returns condition code of the PCC instruction
  */
-static int s390_pcc(unsigned long func, void *param)
+static inline int s390_pcc(unsigned long func, void *param)
 {
-    register unsigned long r0 __asm__("0") = (unsigned long)func;
-    register unsigned long r1 __asm__("1") = (unsigned long)param;
-
-    __asm__ volatile (
-        "0: .long   %[opc] << 16\n"
-        "   brc 1,0b\n"
-        :
-        : [fc] "d" (r0), [param] "a" (r1), [opc] "i" (0xb92c)
-        : "cc", "memory");
-
-    return 0;
+	register unsigned long r0 __asm__("0") = (unsigned long)func;
```

Comment:
Please use OCK coding style, i.e. 4 blanks instead of tabs.

Reply:
Fixed.